content/news/2018-07-11-navcoin-core-4-3-0-release-candidate.md

```diff
@@ -0,0 +1,61 @@
+---
+layout: news
+generated_by_cms: true
+title: NavCoin Core 4.3.0 Release Candidate
+author: Craig MacGregor
+date: '2018-07-11T19:11:27+12:00'
+feature_image: /images/uploads/11-07-2018.png
+categories:
+  - Nav News
+---
+This week brings your chance to test and comment on the release candidate for NavCoin Core 4.3.0. This update introduces the OpenAlias feature for NavCoin core, adds wallet support for bootstrapping, and delivers some GUI fixes.
+
+This week brings your chance to test and comment on the release candidate for NavCoin Core 4.3.0. This update introduces the OpenAlias feature for NavCoin core, adds wallet support for bootstrapping, and delivers some GUI fixes.
```

Comment:
is it intended that this paragraph is duplicated?

Reply:
Yeah, for some reason the templating engine pulls out the first paragraph as the text for the /news page and omits it from the main article page, so we have to duplicate it like this so it displays in both locations.

Reply:
if you check out the preview, it looks fine, it's not duplicated on the article page.